doc/rtd/topics/faq.rst

```diff
@@ -3,19 +3,208 @@
 FAQ
 ***
 
-Getting help
-============
+How do I get help?
+==================
 
 Having trouble? We would like to help!
 
+- First go through this page with answers to common questions
 - Use the search bar at the upper left to search these docs
 - Ask a question in the ``#cloud-init`` IRC channel on Freenode
 - Join and ask questions on the `cloud-init mailing list <https://launchpad.net/~cloud-init>`_
-- Find a bug? `Report bugs on Launchpad <https://bugs.launchpad.net/cloud-init/+filebug>`_
+- Find a bug? Check out the :ref:`reporting_bugs` topic for
+  how to report one
 
+Where are the logs?
+===================
 
-Media
-=====
+Cloud-init uses two files to log to:
+
+- `/var/log/cloud-init-output.log`: captures the output from each stage of
+  cloud-init when it runs
+- `/var/log/cloud-init.log`: very detailed log with debugging output,
+  detailing each action taken
+- `/run/cloud-init`: contains logs about how cloud-init decided to enable or
+  disable itself, as well as what platforms/datasources were detected. These
+  logs are most useful when trying to determine what cloud-init ran or did not
+  run.
+
+Be aware that each time a system boots, new logs are appended to the files in
+`/var/log`. Therefore, the files may have more than one boot worth of
+information present.
+
+When reviewing these logs look for any errors or Python tracebacks to check
+for any errors.
+
+Where are the configuration files?
+==================================
+
+Cloud-init config is provided in two places:
+
+- `/etc/cloud/cloud.cfg`
+- `/etc/cloud/cloud.cfg.d/*.cfg`
+
+These files can define the modules that run during instance initialization,
+the datasources to evaluate on boot, and other settings.
+
+Where are the data files?
+=========================
+
+Inside the `/var/lib/cloud/` directory there are two important subdirectories:
```

Comment:
It's probably worth calling out /run/cloud/* too which contains initial system detection logs such as ds-identify.log cloud-init-generator.log and instance-data*json files which cloud-init obtains from crawling the discovered datasource.

Reply:
I don't have a /run/cloud/ on any system, but if you meant /run/cloud-init, it is already mentioned in the logs section. I don't consider it data files per say

Reply:
yeah you are right, I had meant /run/cloud-init. but I had tunnel vision on this section. Resolved.